### DISCUSSION TITLE
Project active?

### DISCUSSION THREAD

**Discussion by goettl79** [hubtype/botonic#2293] [General]
Hi,

I think the project looks great, but I can see the contributions dropped over the last months. Does anybody know, if there was something like a main innovator left the project or hubetype decided to go "private" or dual licensing? Would be interesting to know.

Regards
Georg

**Comment by vanbasten17** [comment]
Yes, @goettl79 the project is still active and will continue to be open-source! We have done recent releases! Happy coding!
